bazel/constants.go

```diff
@@ -14,4 +14,11 @@ const (
 	// Nil-data/Empty SHA-256 data
 	EmptySha  = "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855"
 	EmptySize = int64(0)
+
+	// GRPC Server connection-related setting limits (defaults)
+	MaxSimultaneousConnections = 0 // limits total simultaneous connections via the Listener
+	MaxRequestsPerSecond       = 0 // limits total incoming requests allowed per second
+	MaxRequestsBurst           = 0 // allows this many requests in a burst faster than MaxRPS average
+	MaxConcurrentStreams       = 0 // limits concurrent streams _per client_
+	MaxConnIdleMins            = 1 // limits open idle connections before the server closes them
```

Comment:
these first 4 constants already exist in bazel/cas/constants.go, we should remove those or move MaxConnIdleMins into there

Reply:
oh... unless we're attempting to have different defaults for the CAS vs for the scheduler...

Reply:
yep - leaving the default unlimited and having scheduler use that (was hardcoded in binaries/main.go previously)